client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotcontents.yaml

```diff
@@ -19,7 +19,42 @@ spec:
     singular: volumegroupsnapshotcontent
   scope: Cluster
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - description: Indicates if the individual snapshots is ready to be used to restore
```

Comment:
`individual snapshots in the group are ready to be used to restore a group of volumes`

Reply:
thanks, fix it